packages/library/src/providers/CFDProvider/CFDProvider.tsx

```diff
@@ -25,8 +27,9 @@ export const useCFDContext = () => {
     return context;
 };
 
-export const CFDProvider = ({ children }: { children: React.ReactNode }) => {
+export const CFDProvider = ({ children }: { children: ReactNode }) => {
```

Comment:
```suggestion
export const CFDProvider = ({ children }: PropsWithChildren) => {
```
Don't forget to import 😃 ☝🏻